src/js/core/widget/core/Page.js

```diff
@@ -456,9 +457,12 @@
 					}
 
 					if (footer) {
-						bottom = utilsDOM.getElementHeight(footer);
-						contentStyle.marginBottom = bottom + "px";
-						contentStyle.paddingBottom = (-bottom) + "px";
+						footerRect = footer.getBoundingClientRect();
+						if (footerRect) {
```

Comment:
According to w3c (https://www.w3.org/TR/2009/WD-cssom-view-20090804/#the-getclientrects-and-getboundingclient) this check is unnecessary - the getBoundingClientRect method always returns an ClientRect object

Reply:
Fixed.